Dockerfile

```diff
@@ -4,10 +4,13 @@ RUN apk --no-cache --no-progress upgrade && \
     apk --no-cache --no-progress --repository http://dl-3.alpinelinux.org/alpine/edge/testing/ --allow-untrusted add dante-server && \
     rm -rf /tmp/*
 
+HEALTHCHECK --interval=5m --timeout=20s --start-period=1m \
+  CMD if test $( curl -m 10 -s https://api.nordvpn.com/vpn/check/full | jq -r '.["status"]' ) = "Protected" ; then exit 0; else nordvpn connect ${CONNECT} ; exit $?; fi

```

Comment:
Does the container already have `jq` built in?  I haven't tested it.

Also, why not run `start.sh`?

Reply:
The container does have `jq` built in. I am not sure what you mean by running `start.sh`. I am very unexperienced with docker and saw that the health checks were not being performed when I booted my container.

Reply:
Cool.

My concern is that `HEALTHCHECK` is not supposed to try to recover, unless I am mistaken.

https://docs.docker.com/engine/reference/builder/#healthcheck

Reply:
Updated in the latest commit